MANIFEST.in

```diff
@@ -2,6 +2,6 @@ include README.md
 include py/visdom/VERSION
 include py/visdom/py.typed
 include py/visdom/*.pyi
-recursive-include py/visdom/static/*
+recursive-include py/visdom/static *
```

Comment:
What is the purpose of this change?

Reply:
Actually, when I tried using the former syntax and run the command `python setup.py install` to test, MathJax folder was missing in the js. Hence, researched a bit and found something what the latter statements, and _hallelujah_, everything worked. But as you have mentioned that you will not allow just commit, then this change can be revoked.
Still, this change won't harm the sequence for sure.